spec/rmagick/image_list/collect_spec.rb

```diff
@@ -1,3 +1,4 @@
+# rubocop:disable Style/CollectionMethods
```

Comment:
No a fan of disabling rules per file. Is it not possible to enable the rule here?

Reply:
@dlemstra In principle, I agree, but these are a weird situation. This is actually testing the method that Rubocop doesn't want us to use. We're testing `ImageList#collect`, but it wants us to use `ImageList#map`.